examples/ansible/enable_root_ssh_login.yml

```diff
@@ -0,0 +1,42 @@
+# Ansible Playbook: Enable Root Login via SSH on ALP Dolomite
+# Description: This playbook automates the process of enabling root login via SSH on ALP Dolomite.
```

Comment:
Perhaps explain here why we need to do this? And also what sort of root login this enables? E.g. is it enabling root login via password? Or via SSH Key? Or both? And do we need both to be enabled?